makefile.shade

```diff
@@ -295,7 +298,7 @@ var RC_FILES = '${FindAllFiles("Resource.rc", BOOTSTRAPPER_EXE_NAME, BOOTSTRAPPE
             {
                 File.Delete(targetShFile);
             }
-            
+
```

Comment:
Ever since installing [Mads' Trailing Whitespace Visualizer](https://visualstudiogallery.msdn.microsoft.com/a204e29b-1778-4dae-affd-209bea658a59) I can't avoid cleaning it up :)